.github/workflows/format-check.yml

```diff
@@ -1,7 +1,13 @@
 # Copyright Kani Contributors
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 name: Kani Format Check
-on: pull_request
+on:
```

Comment:
I think this is unnecessary on push, but I guess I don't object strongly

Reply:
One can say that about all our workflows. I was wondering if we should just be consistent.